roles/confluent.variables_handlers/defaults/main.yml

```diff
@@ -60,14 +60,25 @@ ssl_self_signed_ca_password: capassword123
 # Zookeeper Variables
 zookeeper_user: "{{zookeeper_default_user}}"
 zookeeper_group: "{{zookeeper_default_group}}"
+zookeeper_sasl_protocol: "{{sasl_protocol if sasl_protocol == 'kerberos' else 'none'}}"
+# Variable only used by Kafka for the time being, health checks/components will use 2181
+zookeeper_client_port: "{{'2182' if zookeeper_ssl_enabled|bool else '2181'}}"
+zookeeper_ssl_enabled: "{{ssl_enabled}}"
```

Comment:
default to false for the time being, should be like this, but wouldnt be backwards compatible

Reply:
need to add documentation